### DISCUSSION TITLE
What is the reason, the web application is not built with Expo?

### DISCUSSION THREAD

**Discussion by goulashsoup** [prince272/nextsolution#1] [Q&A] [answered]
@prince272 AFAIK Expo can be developed for Web, iOS and Android, so why not use it for web as well?

Seems duplicate work to maintain the web version separately...

**Comment by prince272** [comment, marked as answer]
Even with Expo's cross-platform capabilities, a separate web version might be needed because Expo uses React Native, which is optimized for mobile apps, while Next.js uses React and is tailored for web applications. This ensures each platform gets the best performance and user experience.
